src/main/java/com/metaphorce/security/utils/JwtUtils.java

```diff
@@ -0,0 +1,38 @@
+package com.metaphorce.security.utils;
+
+import com.auth0.jwt.JWT;
+import com.auth0.jwt.algorithms.Algorithm;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.security.core.Authentication;
+import org.springframework.security.core.userdetails.User;
+import org.springframework.stereotype.Component;
+
+import java.util.Date;
+
+/**
+ * Utility class for generating and handling JWT tokens.
+ */
+@Component
+public class JwtUtils {
+
+    @Value("${app.jwtSecret}")
+    private String jwtSecret;
+
+    @Value("${app.jwtExpirationMs}")
+    private int jwtExpirationMs;
+
+    /**
+     * Generates a JWT token for the authenticated user.
+     *
+     * @param authentication the authentication object containing user details.
+     * @return a JWT token as a string.
+     */
+    public String generateToken(Authentication authentication) {
+        User principal = (User) authentication.getPrincipal();
+        return JWT.create()
+                .withSubject(principal.getUsername())
+                .withIssuedAt(new Date())
```

Comment:
Use LocalDate instead of Date because Date is deprecated

Reply:
![image](https://github.com/j0rgel0/service-metaphorce/assets/102724257/73840882-3a4a-4938-bf97-ae770867015d)
The java-jwt library currently does not support LocalDateTime directly for setting JWT claims due to issues with time zones and compatibility with the JWT specification, which primarily uses Instant or java.util.Date